### DISCUSSION TITLE
Hope to support AnyContainer function being turned on by default

### DISCUSSION THREAD

**Discussion by ApliNi** [Jikoo/OpenInv#241] [Ideas]
Hi, I noticed that the AnyContainer permission has been removed and replaced with a command to turn it on and off. But I think this makes the use of this feature more complicated, especially for new players, and it may be appropriate to make it a default-enabled feature. Could you consider adding a configuration to make it enabled by default? This will not conflict with the change in permissions.

**Comment by Jikoo** [comment]
OI doesn't persist toggle state any more, so you'd need an addon to do that. It would basically be the inverse of OITogglePersist, tracking players who have opted out of a toggle and turning it on for everyone else.
